extensions/amp-audio/0.1/amp-audio.js

```diff
@@ -92,6 +101,16 @@ export class AmpAudio extends AMP.BaseElement {
     this.element.appendChild(audio);
     this.audio_ = audio;
 
+    listen(this.audio_, 'playing', () => this.audioPlaying_());
+  }
+
+  /** @override */
+  layoutCallback() {
```

Comment:
Retaining constrution of the element in `layoutCallback ()` if layout is not `NODISPLAY`